tutor/src/components/task-step/step-with-reading-content.cjsx

```diff
@@ -43,7 +43,11 @@ ReadingStepContent = React.createClass
 
   getContentChapterSection: ->
     {chapter_section, related_content} = TaskStepStore.get(@props.id)
-    (related_content && related_content[0]) || chapter_section || []
+    (
+      related_content &&
+      related_content[0] &&
+      related_content[0].chapter_section
+    ) || chapter_section || []
 
```

Comment:
fyi - I like using lodash [get](https://lodash.com/docs/#get) for this since it deals with null values, this could be: 

`get(related_content, '[0].chapter_section, chapter_section || [])`